src/common/components/PageLayout.jsx

```diff
@@ -27,7 +29,11 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'column',
   },
   desktopDrawer: {
-    width: theme.dimensions.drawerWidthDesktop,
+    width: (props) => (props.miniVariant ? `calc(${theme.spacing(7)} + 1px)` : theme.dimensions.drawerWidthDesktop),
```

Comment:
`calc(${theme.spacing(7)} + 1px)` doesn't seem right. I know there is a version of a collapsed sidebar. Can we somehow use the size from there?

Reply:
Or maybe even use it as is instead of implementing our own thing.

https://mui.com/material-ui/react-drawer/#mini-variant-drawer

Reply:
That's exactly where that was taken from

Reply:
It's not a component, it's an example how to do the mini variant.

Reply:
Oh I didn't realize that.